squad/api/rest.py

```diff
@@ -571,6 +572,25 @@ def unsubscribe(self, request, pk=None):
         data = {"email": subscriber_email}
         return Response(data, status=status.HTTP_200_OK)
 
+    @action(detail=True, methods=['get', 'post'], suffix='compare_builds', permission_classes=[AllowAny])
+    def compare_builds(self, request, pk=None):
+        builds_to_compare = {
+            "first": request.data.get("first", None) if request.method == 'POST' else request.query_params.get("first", None),
```

Comment:
baseline? This is the name which we use in ProjectStatus so I think it would be consistent to use here as well.

Reply:
Agree.